docs/data-builder-dummy-data.md

```diff
@@ -0,0 +1,39 @@
+# Supplying a dummy dataset
+
+---8<-- 'includes/data-builder-danger-header.md'
+
+Because OpenSAFELY doesn't allow direct access to individual patient
+records, researchers must use a **dummy dataset** for developing their analytic
+code on their own computer.
+
+A dummy dataset needs to be similar enough to a real dataset to allow
+the analysis scripts to run successfully, but it must be randomly (or
```

Comment:
For consistency, how about...

```suggestion
[the project pipeline](actions-pipelines.md) to run successfully, but it must be randomly (or
```

Reply:
In addition to @iaindillingham's suggestion, I'd probably also emphasise the "not contain data relating to real patients" at the end of this line.

At the least, I'd break this last clause out into its own sentence. It's very important, but is at the end. Maybe even with an admonition like `!!! warning`? [^1]

[^1]: This might reasonably be `!!! danger`. But searching through the Markdown source, `!!! danger` has only been used within the Data Builder docs so far. Therefore there may be similarly-themed text where we're not using `!!! danger`, and for consistency, I'd leave for now, and maybe address that later.